app/qml/main.qml

```diff
@@ -270,16 +270,16 @@ ApplicationWindow {
           mapThemesPanel.visible = true
           stateManager.state = "misc"
         }
-        onMyLocationClicked: map.centerToPosition()
 
-        onMyLocationHold: {
-            __appSettings.autoCenterMapChecked = !__appSettings.autoCenterMapChecked
-            showMessage( __appSettings.autoCenterMapChecked ?  qsTr("GPS auto-center mode on") : qsTr("GPS auto-center mode off") )
+        onMyLocationClicked: {
```

Comment:
@alexbruy for the sake of keeping the UX same as before, can you also add a slot for `onMyLocationHold` that would do exactly the same as `onMyLocationClicked`? Thanks!